readme.adoc

```diff
@@ -18,51 +18,52 @@ Releases are available here: https://github.com/neo4j-contrib/neo4j-graph-algori
 
 === Centralities
 
-These algorithms determine the importance of distinct nodes in a network.
+These algorithms determine the importance of distinct nodes in a network:
 
 * Page Rank (`algo.pageRank`)
 * Betweenness Centrality (`algo.betweenness`)
 * Closeness Centrality (`algo.closeness`)
 
-=== Community Detection
+=== Community detection
 
-These algorithms evaluate how a group is clustered or partitioned, as well as its tendency to strengthen or break apart.
+These algorithms evaluate how a group is clustered or partitioned, as well as its tendency to strengthen or break apart:
 
 * Louvain (`algo.louvain`)
 * Label Propagation (`algo.labelPropagation`)
 * (Weakly) Connected Components (`algo.unionFind`)
 * Strongly Connected Components (`algo.scc`)
 * Triangle Count / Clustering Coefficient (`algo.triangleCount`)
 
-=== Path Finding
+=== Path finding
 
-These algorithms help find the shortest path or evaluate the availability and quality of routes.
+These algorithms help find the shortest path or evaluate the availability and quality of routes:
 
 * Minimum Weight Spanning Tree (`algo.mst`)
 * All Pairs- and Single Source - Shortest Path (`algo.shortestPath`, `algo.allShortestPaths`)
 
 // end::algorithms[]
 
-These procedures work either on the whole graph or on a subgraph filtered by label and relationship-type.
+These procedures work either on the whole graph, or on a subgraph filtered by label and relationship-type.
 You can also use link:#cypher-projection[filtering and projection using Cypher queries].
 
-*We'd love your feedback*, so please try out these algorithms and let us know how well they work for your use-case.
-Also please note things that are missing from the installation instructions or documentation.
-
-Please raise https://github.com/neo4j-contrib/neo4j-graph-algorithms/issues[GitHub issues] for anything you encounter or join the http://neo4j.com/developer/slack[neo4j-users Slack group] and ask in the `#neo4j-graph-algorithm` channel.
-
 == Installation
 
-Download `graph-algorithms-algo-*.jar` from https://github.com/neo4j-contrib/neo4j-graph-algorithms/releases[the matching release] and copy it into your `$NEO4J_HOME/plugins` directory.
+Download `graph-algorithms-algo-[version].jar` from https://github.com/neo4j-contrib/neo4j-graph-algorithms/releases[the matching release] and copy it into your `$NEO4J_HOME/plugins` directory.
 
-Because the algorithms use the lower level Kernel API to read from and write to Neo4j you will also have to enable them in the configuration (for security reasons):
+Because the algorithms use the lower level Kernel API to read from, and to write to Neo4j, for security purposes you will also have to enable them in the configuration:
 
-.Add to $NEO4J_HOME/conf/neo4j.conf
+. Add the following to your _$NEO4J_HOME/conf/neo4j.conf_ file:
++
 ----
-dbms.security.procedures.unrestricted=algo.*
+dbms.security.procedures.unrestricted=algo.<name>
```

Comment:
actually we need to keep the star as otherwise people will have a very frustrating experience having to enable each algorithm individually.